build/dev-env.sh

```diff
@@ -47,21 +47,21 @@ fi
 
 HELM_VERSION=$(helm version 2>&1 | grep -oE 'v[0-9]+\.[0-9]+\.[0-9]+') || true
 if [[ ${HELM_VERSION} < "v3.0.0" ]]; then
-  echo "Please upgrade helm to v3.0.0 or higher"
+  echo "Please upgrade helm to v3.9.0 or higher"
   exit 1
 fi
 
 KUBE_CLIENT_VERSION=$(kubectl version --client --short 2>/dev/null | grep Client | awk '{print $3}' | cut -d. -f2) || true
 if [[ ${KUBE_CLIENT_VERSION} -lt 14 ]]; then
```

Comment:
Ditto 

This condition just for 1.14+

Reply:
I can do this change. But some people may not have updated. So I left it at that. This way if it fails, then suggestion is to install latest else if it passes, they can continue to use whatever version they have and upgrade as per their timeline. I will wait for your comment and change or keep it, based on your comment.

Reply:
Changed as per suggestion